doc/man3/flux_shell_task_get_info.adoc

```diff
@@ -0,0 +1,53 @@
+flux_shell_task_get_info(3)
+===========================
+:doctype: manpage
+
+NAME
+----
+flux_shell_task_get_info, flux_shell_task_info_unpack - interfaces for fetching task info
+
+SYNOPSIS
+--------
+ #include <flux/shell.h>
+
+ int flux_shell_task_get_info (flux_shell_task_t *task, 
+                               char **json_str);
+
+ int flux_shell_task_info_unpack (flux_shell_task_t *task, 
+                                  const char *fmt, ...);
+
+DESCRIPTION
+-----------
+
+Returns task info either as a json string (specified below) or 
+using Jansson-style parameters.  The structure of the former is:
+
+ "localid":i,
+ "rank":i,
+ "state":s,
+ "pid":I,
+ "wait_status":i,
+ "exitcode":i,
+ "signaled":i
+ 
+RETURN VALUES
+-------------
+
+Neither function is implemented yet.
+
```

Comment:
```suggestion
RETURN VALUE
-------------

These functions return 0 on success, -1 on error with `errno` set.

ERRORS
------
EINVAL::
One or more arguments were invalid

```

Reply:
My bad --- found these in task.c.  Done.